libsrc/c64/soft80_color.s

```diff
@@ -0,0 +1,109 @@
+;
+; unsigned char __fastcall__ textcolor (unsigned char color);
+; unsigned char __fastcall__ bgcolor (unsigned char color);
+; unsigned char __fastcall__ bordercolor (unsigned char color);
+;
+
+        .export         soft80_textcolor, soft80_bgcolor, soft80_bordercolor
+        .export         __textcolor, __bgcolor
+
+        .importzp       tmp1, tmp2
+
+        .import         soft80_checkchar
```

Comment:
As far as I see this line is unnecesarry - and otherwise wouldn't work anyway as there's no export.